.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI/CD
+
+on:
```

Comment:
Could we update the triggers to match this for consistency?
https://github.com/puppetlabs/puppet_litmus/blob/05488913fe3dd0226793a3a0c7c8d233e02e18eb/.github/workflows/ci.yml#L3

Reply:
The design uses a template, so the `on` triggers are intended to be in each workflow file.

CI workflow automatically determines if images should be pushed to the docker repository based on the branch (main) using conditional steps in the template. Changing the on to include `workflow_dispatch` serves to purpose, and removing `push` would mean new images never get built / pushed until the nightly job.

Manual workflow includes the ability to set and pass inputs to the template, which the CI workflow does not (by design). Defaults are set to the same as CI.